packages/rulesets/src/spectral/functions/default-in-enum.ts

```diff
@@ -0,0 +1,21 @@
+//This rule applies when the value specified by the default property
+//does not appear in the enum constraint for a schema.
+const defaultInEnum = (swaggerObj:any, _opts:any, paths:any) => {
+  const defaultValue = swaggerObj.default;
+  const enumValue: string[] = swaggerObj.enum;
+  if (swaggerObj === null || typeof swaggerObj !== 'object') {
+    return [];
+  }
+  const path = paths.path || paths.target || [];
+
+  if (enumValue && !enumValue.includes(defaultValue)) {
+    return [{
```

Comment:
what if defaultValue is not defined ?

Reply:
It should also be reported as a violation.

Reply:
I checked donet code, if either default value or enum is undefined, it is valid .